judge/views/submission.py

```diff
@@ -274,7 +274,7 @@ def _get_queryset(self):
         if self.selected_languages:
             queryset = queryset.filter(language__in=Language.objects.filter(key__in=self.selected_languages))
         if self.selected_statuses:
-            queryset = queryset.filter(result__in=self.selected_statuses)
+            queryset = queryset.filter(Q(result__in=self.selected_statuses) | Q(status__in=self.selected_statuses))
```

Comment:
the `Or` could be heavy, could you optimize it? Since those two result & status can be separate

Reply:
We allow selecting multiple statuses, so I don't think it's possible